website/docs/admin_hosts_houdini.md

```diff
@@ -5,16 +5,18 @@ sidebar_label: Houdini
 ---
 ## General Settings
 ### JOB Path
-you can add your studios preffered JOB Path, JOB value will be checked and updated on file save and open.
-Disableing this option will effectivly turn off this feature.
+Specify a studio-wide `JOB` path.<br>
```

Comment:
A bit of semantics - but it's not necessarily studio-wide. It's a project setting, so project specific. Plus it can be dynamic per asset or task - so whether 'studio-wide' sounds correct for that I'm not sure. It kind of sounds like a studio specific static path. But maybe it's just my interpretation of the sentence.

Reply:
`<br>` doesn't generate an empty line, if it's not a big deal no problem. 
![image](https://github.com/ynput/OpenPype/assets/20871534/e73b3f4f-7701-42b4-9793-7abf34901ff1)

without `<br>`
![image](https://github.com/ynput/OpenPype/assets/20871534/e701e3e7-b34a-4cd3-8fb6-116178b8bac4)

Reply:
It does add the empty line if you add an extra enter like in my suggestion, right?

Reply:
I see - I personally prefer the second style anyway.